include/libc/stdint.h

```diff
@@ -0,0 +1,27 @@
+#ifndef STDINT_H
+#define STDINT_H
+
+typedef signed int intptr_t;
+typedef unsigned int uintptr_t;
+
+#define INT8_MIN    (-0x80)
+#define INT16_MIN   (-0x8000)
+#define INT32_MIN   (-0x80000000)
+#define INT64_MIN   (-0x8000000000000000)
+
+#define INT8_MAX    0x7f
+#define INT16_MAX   0x7fff
+#define INT32_MAX   0x7fffffff
+#define INT64_MAX   0x7fffffffffffffff
```

Comment:
the repo uses uppercase hexadecimal everywhere. idk if this should apply to a "standard" header